opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -1533,51 +1532,36 @@ namespace Opm {
 
 
     public:
-        int ebosCompToFlowPhaseIdx( const int compIdx ) const
-        {
-            assert(compIdx < 3);
-            const int compToPhase[ 3 ] = { Oil, Water, Gas };
-            return compToPhase[ compIdx ];
-        }
-
-        int flowToEbosPvIdx( const int flowPv ) const
```

Comment:
The same function `flowToEbosPvIdx` from WellInterface should also be removed.

Reply:
flowToEbosPvIdx is removed from WellInterface